### PR TITLE
Release v2.0.2.

### PR DESCRIPTION
**Pull request by theacodes**
`git log v2.0.1..HEAD | grep 'Merge pull request ' | awk -F ' ' '{print $4 $5}' | sort` yields:
#446 - Python 3 fixes for step2_exchange.
#452 - flask_util requests new credentials when expired.
#457 - Typo
#460 - Coverage
#462 - Coverage
#464 - Coverage
#467 - Coverage
#477 - Coverage
#478 - Moved openers
#479 - Coverage
#481 - Coverage
#482 - Coverage
#483 - Coverage
#486 - Coverage
#490 - Coverage
#492 - Coverage
#493 - Coverage
#495 - Coverage
#496 - Coverage
#498 - Fix flask_util session thrashing
#500 - Fix flask_util docstrings
#457, #478, and #500 were not included in release notes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v2.0.2
+
+* Fix issue where `flask_util.UserOAuth2.required` would accept expired credentials (#452).
```
Comment (dhermes) [inline CHANGELOG.md:5]: 80 chars? Who cares?

**Comment by dhermes** [comment]
LGTM. Do we need to wait on @nathanielmanistaatgoogle?

File: CHANGELOG.md (hunk 1 above)
Comment (theacodes) [inline CHANGELOG.md:5]: I don't care.

**Comment by jonparrott** [comment]
2/3 is good enough for me, especially since this is a patch release.

**Comment by dhermes** [comment]
@jonparrott Don't forget to create the tag.

**Comment by jonparrott** [comment]
Thanks. Done. :)

**Pull request merged**